lispy.el

```diff
@@ -8579,23 +8584,18 @@ When ARG is non-nil, unquote the current string."
     (define-key map (kbd "M-?") 'lispy-convolute-sexp)
     (define-key map (kbd "M-S") 'lispy-split)
     (define-key map (kbd "M-J") 'lispy-join)
-    (define-key map (kbd "{") 'lispy-braces)
-    (define-key map (kbd "}") 'lispy-brackets)
-    (define-key map (kbd "]") 'lispy-forward)
-    (define-key map (kbd "[") 'lispy-backward)
-    (define-key map (kbd "M-(") 'evil-cp-wrap-next-round)
-    (define-key map (kbd "M-{") 'evil-cp-wrap-next-curly)
-    (define-key map (kbd "M-}") 'evil-cp-wrap-next-square)
-    (define-key map (kbd "<") 'evil-cp-<)
-    (define-key map (kbd ">") 'evil-cp->)
-    (define-key map (kbd "y") 'lispy-new-copy)
```

Comment:
I agree it is not convenient. To make it consistent with both paredit and cleverparens, what do you think bout:

```
     (define-key lispy-mode-map (kbd "C-<") 'evil-cp-<)
     (define-key lispy-mode-map (kbd "C->") 'evil-cp->)
```